src/network/protocol/messages/lib.zig

```diff
@@ -15,6 +16,7 @@ pub const MessageTypes = enum {
     Getblocks,
     Ping,
     Pong,
+    Getdata,
```

Comment:
you will need to rebase and make this lowercase

Reply:
ok